projects/packages/promote-posts/composer.json

```
@@ -0,0 +1,46 @@
+{
+	"name": "automattic/jetpack-promote-posts",
+	"description": "Attract high-quality traffic to your site using Promoted Posts.",
+	"type": "jetpack-library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"yoast/phpunit-polyfills": "1.0.4",
+		"automattic/jetpack-changelogger": "@dev"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"phpunit": [
+			"./vendor/phpunit/phpunit/phpunit --colors=always"
+		],
+		"test-coverage": [
+			"php -dpcov.directory=. ./vendor/bin/phpunit --coverage-clover \"$COVERAGE_DIR/clover.xml\""
+		],
+		"test-php": [
+			"@composer phpunit"
+		],
+		"build-production": "echo 'Add your build step to composer.json, please!'",
+		"build-development": "echo 'Add your build step to composer.json, please!'"
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"extra": {
+		"branch-alias": {
+			"dev-trunk": "0.1.x-dev"
+		},
+		"textdomain": "jetpack-promote-posts"
```

```suggestion
		"textdomain": "jetpack-promote-posts",
		"version-constants": {
			"::PACKAGE_VERSION": "src/class-promote-posts.php"
		}
```

This will allow us to automate that number incrementation.

---

Cool, let's update the scaffolding to the latest pattern.

---

> Cool, let's update the scaffolding to the latest pattern.

On my radar.

---

I believe it would be handled by the work we had started doing on automating the mirror repo creation; those things would get added by the generate script when the mirror repo is created.
Maybe we should "just" resume the work on that :)